subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/TestTaskIntegrationTest.groovy

```diff
@@ -282,6 +282,152 @@ class TestTaskIntegrationTest extends JUnitMultiVersionIntegrationSpec {
         extraArgs << [[], ["--tests", "MyTest"]]
     }
 
+    def "options set prior to setting same test framework will warn and have no effect"() {
+        ignoreWhenJUnitPlatform()
```

Comment:
I missed the need to run this class using the `integMultiVersionTest` locally until it failed on CI.  Need to ensure proper testing framework is being used.